src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,17 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, getProperties(authClient, options, LoginRouter));
+    _.extend(this, LoginRouter.prototype.Events);
```

Comment:
_.extend can take multiple args, so we can use:
```javascript
// Note: I think it's nicer to extend events first, just in case later on we actually want to listen to it in the getProperties method.
_.extend(this, LoginRouter.prototype.Events, getProperties(stuff));
```